discord/asset.py

```diff
@@ -132,15 +134,15 @@ def _from_guild_icon(cls, state, guild, *, format=None, static_format='webp', si
         if format is None:
             format = 'gif' if guild.is_icon_animated() else static_format
 
-        return cls(state, 'https://cdn.discordapp.com/icons/{0.id}/{0.icon}.{1}?size={2}'.format(guild, format, size))
+        return cls(state, '/icons/{0.id}/{0.icon}.{1}?size={2}'.format(guild, format, size))
 
 
     def __str__(self):
-        return self._url if self._url is not None else ''
+        return self.BASE + self._url if self._url is not None else ''
```

Comment:
is it really `self.BASE`?

Reply:
https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables